Lib/unittest/__init__.py

```diff
@@ -78,3 +77,14 @@ def load_tests(loader, tests, pattern):
     # top level directory cached on loader instance
     this_dir = os.path.dirname(__file__)
     return loader.discover(start_dir=this_dir, pattern=pattern)
+
+
+def __dir__():
+    return globals().keys() | {'IsolatedAsyncioTestCase'}
+
+def __getattr__(name):
+    if name == 'IsolatedAsyncioTestCase':
```

Comment:
Please add a comment explaining why asyncio is imported lazily.

Maybe document also that this code implements lazy import of async_case :-) It may not be obvious at the first read.